### PR TITLE
[Behat] Added default behat.yml extending behat.yml.dist

### PR DESCRIPTION
**Pull request by pamil**
Added `behat.yml` boilerplate to make customizing Behat even easier!

### CODE REVIEW

File: behat.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+imports: ["behat.yml.dist"]
```
Comment (pjedrzejewski) [inline behat.yml]: Any reason to use double quotes?

Comment (pamil) [inline behat.yml]: Seems cleaner to me than:
```yaml
imports: [behat.yml.dist]
```

**Comment by michalmarcinkowski** [comment]
:+1:

**Comment by pjedrzejewski** [comment]
:+1:

**Pull request merged**